vibi-dpu/src/github/config.rs

```diff
@@ -85,6 +99,38 @@ async fn get_all_pages(next_url: Option<String>, access_token: &str, params: &Op
     return all_values;
 }
 
+async fn deserialize_paginated_webhook_response(response_opt: Option<Response>) -> PaginatedResponse {
```

Comment:
This function seems to be generic and it probably handles the job of `deserialize_paginated_response` for everyone. 

So we can rename this function to the generic version and use it for deserializing the response for the repositories API call as well. That way, we'd be able to resolve the comment in PR #77 as well.

Reply:
[Here](https://github.com/Alokit-Innovations/vibi-dpu/pull/77#discussion_r1390656227) is the link to the comment in the PR #77.